spec/game_runner_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "spec_helper"
+require_relative "../game_runner"
+
+RSpec.describe "GameRunner" do
+  describe "#run" do
+    it "displays an empty board" do
+      empty_board =
+        <<-SQUIGGLY_HEREDOC
```

Comment:
If you use `<<~`, you'll probably find it easier to deal with alignment things.

Reply:
💡